templates/webapi/test/logfile.html.ep

```diff
@@ -5,11 +5,13 @@
 %= asset $_ for qw(test_result.js anser.js ansi-colors.css)
 % end
 % content_for 'ready_function' => begin
-    loadEmbeddedLogFiles();
+    filterEmbeddedLogFiles();
 % end
 
 % my $url = url_for('test_file', testid => $testid, filename => $filename);
 <div class="corner-buttons" style="margin-top: -5px;">
+<span id="search-info"></span>
+<input id="search-log-file" placeholder="substring or /regex/i" type="search">
```

Comment:
this could also be

```suggestion
<span id="filter-info"></span>
<input id="filter-log-file" placeholder="substring or /regex/i" type="search">
```